docs/01.GETTING-STARTED.md

```diff
@@ -4,64 +4,70 @@ Currently, only Ubuntu 14.04+ is officially supported as primary development env
 
 There are several dependencies, that should be installed manually. The following list is required for building:
 
-- `gcc` higher than `4.8.2`
+- `gcc` >= `4.8.2`
   - native
```

Comment:
Now that we're a pure C99 codebase (not even relying on GNU extensions) it should be possible to build JerryScript with much older GCC versions/pretty much any C99-compliant compiler.